docs/src/content/docs/es/index.mdx

```diff
@@ -1,26 +1,43 @@
 ---
-title: Bienvenido, mundo
-description: Starlight te ayuda a construir sitios web de documentación hermosos y de alto rendimiento con Astro.
+title: Starlight 🌟 Construye sitios de documentación con Astro
+description: Starlight te ayuda a construir sitios web de documentación hermosos y de alto rendimiento utilizando Astro.
 template: splash
+hero:
+  title: Haz que tus docs brillen con Starlight
+  tagline: Todo lo que necesitas para construir un sitio web de documentación estelar. Rápido, accesible y fácil de usar.
+  image:
+    file: ../../../assets/hero-star.webp
+  actions:
+    - text: Empezar
+      icon: right-arrow
+      variant: primary
+      link: /getting-started/
```

Comment:
Hi, I just noticed something while casually browsing. Shouldn't the link start with `/es/`?

```suggestion
      link: /es/getting-started/
```

Reply:
Oh yes! Very well spotted!

I don't have the same link checking set up here that we have in docs yet, so easy for us to miss this kind of thing. Thanks for looking!